objects/date.js

```diff
@@ -213,6 +235,14 @@ module.exports = class Date extends global.Date {
 		return this;
 	}
 
+	leapYearsBeforeDate() {
```

Comment:
What sort of value is this method returning? Documentation would be appreciated, even expected.